lang/c++/scripts/gen-cppcode.py

```diff
@@ -33,11 +33,11 @@
 
 done = False
 
-typeToC= { 'int' : 'int32_t', 'long' :'int64_t', 'float' : 'float', 'double' : 'double', 
-'boolean' : 'bool', 'null': 'avro::Null', 'string' : 'std::string', 'bytes' : 'std::vector<uint8_t>'} 
+typeToC= { 'int' : 'int32_t', 'long' :'int64_t', 'float' : 'float', 'double' : 'double',
```

Comment:
This file is also modified by https://github.com/apache/avro/pull/532

This will cause merge conflicts.